marklogic-data-hub/src/main/resources/hub-internal-config/security/amps/amps-dhf-delete-job-docs.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
See the comment below about calling this deleteJobs - then just call this file deleteJobs.json